spring/src/main/java/org/axonframework/spring/config/EventHandlerRegistrar.java

```diff
@@ -84,13 +87,11 @@ public void stop(Runnable callback) {
 
     @Override
     public void start() {
-        eventProcessingConfiguration.start();
```

Comment:
These methods were the reason this class implements `SmartLifecycle`. If this isn't necessary anymore, the `SmartLifecycle` methods can also be removed.

Reply:
Good point. I'll remove the `SmartLifecycle` implementation details from this class.